Formula/n/nox.rb

```diff
@@ -20,8 +21,8 @@ class Nox < Formula
   depends_on "python@3.12"
 
   resource "argcomplete" do
-    url "https://files.pythonhosted.org/packages/f0/a2/ce706abe166457d5ef68fac3ffa6cf0f93580755b7d5f883c456e94fab7b/argcomplete-3.2.2.tar.gz"
-    sha256 "f3e49e8ea59b4026ee29548e24488af46e30c9de57d48638e24f54a1ea1000a2"
+    url "https://files.pythonhosted.org/packages/79/51/fd6e293a64ab6f8ce1243cf3273ded7c51cbc33ef552dce3582b6a15d587/argcomplete-3.3.0.tar.gz"
```

Comment:
```suggestion
    url "https://files.pythonhosted.org/packages/79/51/fd6e293a64ab6f8ce1243cf3273ded7c51cbc33ef552dce3582b6a15d587/argcomplete-3.3.0.tar.gz"
```

Not sure if I'm understanding the requirements correctly but should this be using `argcomplete < 3.0` (i.e., 2.1.2)?: https://github.com/wntrblm/nox/blob/5656fcedc31a1ea37d016e4d94d00185330cc528/requirements-conda-test.txt